content/millennium/r4/general-clinical/allergy-intolerance.md

```diff
@@ -21,19 +21,21 @@ Assuming a patient safety or decision support use case, a consumer should avoid
 
 The following fields are returned if valued:
 
-* [Allergy id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
-* [Allergy code](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.code){:target="_blank"}
-* [clinicalStatus](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.clinicalStatus){:target="_blank"}
-* [verificationStatus](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.verificationStatus){:target="_blank"}
-* [Patient with allergy/intolerance](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.patient){:target="_blank"}
-* [Date of allergy onset](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.onset){:target="_blank"}
-* [Date/Time recorded](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.recordedDate){:target="_blank"}
-* [Who recorded](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.recorder){:target="_blank"}
-* [Who reported](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.asserter){:target="_blank"}
-* [Criticality/potential harm](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.criticality){:target="_blank"}
-* [Category (medication, food, environment)](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.category){:target="_blank"}
-* [Adverse reaction (clinical symptoms/manifestation)](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.reaction){:target="_blank"}
-* [Comment](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.note){:target="_blank"}
+* [Allergy Id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Allergy Code](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.code){:target="_blank"}
```

Comment:
After a little more thought the '{:target="_blank") seems unnecessary
Lets model this based on anything of the existing documentations on R4 say - [Condition](https://raw.githubusercontent.com/cerner/fhir.cerner.com/master/content/millennium/r4/general-clinical/condition.md)
Which would mean the following changes as well;

Lets switch the headers on lines 64 and 93 of this file 
```
<%= headers %>
```
to 
```
<%= headers fhir_json: true %>
```
----
Lets add a disclaimer after line 75 and 104 <%= json(:r4_allergy_intolerance_bundle) %>
```
<%= json(:r4_allergy_intolerance_bundle) %>

<%= disclaimer %>
```
----

Reply:
https://github.com/cerner/fhir.cerner.com/pull/321/commits/3a7d707b9ff08376207936f3f54f63fc9a3e01ac